### PR TITLE
Register address for astarbase frontend

### PR DESCRIPTION
**Pull request by niklabh**

### CODE REVIEW

File: src/hooks/useMint.ts [hunk 1]
```diff
@@ -1,18 +1,43 @@
...
   const store = useStore();
   const mintContract = computed(() => store.getters['general/mintContract']);
   const mintContractAddress = computed(() => store.getters['general/mintContractAddress']);
   const account = computed(() => store.getters['general/ethereumAccount']);
+  const substrateAccounts = computed(() => store.getters['general/substrateAccounts']);
+  const substrateAccount = computed(() => store.getters['general/substrateAccount']);
 
   // Todo: Modify the logic once contract has been finalized
-  const mint = () => {
+  const mint = async () => {
```
Comment (Maar-io) [inline src/hooks/useMint.ts:22]: rename `mint` everywhere. It is not minting function

**Comment by Maar-io** [comment]
After register is called, and the Tx is confirmed there should be info message, something like, 
> "Congratulations! You now hold ShidenPass" Use it in the Shiden ecosystem to claim rewards. Note, ShidenPass is valid as long as you are active staker in dApps-Staking"

link to [portal](https://portal.astar.network/#/dapp-staking/discover)

**Comment by Maar-io** [comment]
third bullet: "Register to ShidenPass"
Make all messages easily configurable since we will reuse this for Astar

**Comment by Maar-io** [comment]
After register is successful there should be enabled link to `Mint your Free NFT`

File: public/config/config.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
 {
-  "mintContractAddress": "0x406bC9Fd7D3Bfcb6b0d102FdE21BFF7E5E9F2b9A",
+  "mintContractAddress": "0xB50F050C9Cf804B1A3Fe21663E4f15593c3aaecc",
   "astarBaseContractAddress": "",
```
[the remark this answers is not in the document]
Comment (niklabh) [reply public/config/config.json]: Done

File: src/hooks/useMint.ts (hunk 1 above)
Comment (niklabh) [reply src/hooks/useMint.ts]: Done

**Comment by niklabh** [comment]
> After register is called, and the Tx is confirmed there should be info message, something like,
> 
> > "Congratulations! You now hold ShidenPass" Use it in the Shiden ecosystem to claim rewards. Note, ShidenPass is valid as long as you are active staker in dApps-Staking"
> 
> link to [portal](https://portal.astar.network/#/dapp-staking/discover)

Done. Where to put portal link?

**Comment by niklabh** [comment]
> the first button should say "Connect to Native Wallet"

Done

**Comment by niklabh** [comment]
> third bullet: "Register to ShidenPass" Make all messages easily configurable since we will reuse this for Astar

Done

**Comment by niklabh** [comment]
> After register is successful there should be enabled link to `Mint your Free NFT`

Done

**Comment by Maar-io** [comment]
> > link to [portal](https://portal.astar.network/#/dapp-staking/discover)
> 
> Done. Where to put portal link?

add it to dapps-staking

**Review by Maar-io — APPROVED** [review]
We need some more design updates, but we can mege this

**Pull request merged**